app/assets/stylesheets/components/_escape-link.scss

```
@@ -0,0 +1,25 @@
+.app-c-escape-link {
+  background-color: govuk-colour('white');
+  border-top: 1px solid $govuk-border-colour;
+  bottom: 0;
+  position: sticky;
```

Fallback for browsers that don't support sticky:

```suggestion
  position: fixed;
  position: sticky;
```

---

Good point. This requires a 'width: 100%;' on the container. Updated.